requirements/test-requirements.txt

```diff
@@ -10,16 +10,12 @@ azure-storage-blob>=12.0.0
 azure-identity>=1.6.1
 databricks-cli@git+https://github.com/databricks/databricks-cli.git
 pillow
-shap>=0.40
 plotly
 kaleido
-# Required by mlflow.tensorflow / mlflow.pytorch tests.
-transformers
-tensorboard
 # Required by tuning tests
 hyperopt
 # Required by recipes tests
 ipython
 # Required by automl tests
 # 1.1.0 contains this issue: https://github.com/microsoft/FLAML/issues/871
-flaml!=1.1.0
```

Comment:
This line is not changed, why github shows diff here ?

Reply:
@WeichenXu123 because a newline was inserted.